### PR TITLE
Content disposition fix

### PR DESCRIPTION
**Pull request by denmarkin**

### CODE REVIEW

File: spec/lib/rev/post_inputs_spec.rb [hunk 1]
```diff
@@ -24,6 +24,27 @@
...
+    new_input_location = client.create_input_from_link(link, filename, content_type)
+
+    new_input_location.must_match 'urn:rev:inputmedia:'
+    expected_body = {
+      'url' => link,
+      'filename' => filename,
+      'content_type' => content_type
+    }
+    assert_requested(:post, /.*\/inputs/, :times => 1) do |req|
+      req.headers['Content-Type'] == 'application/json'
```
Comment (amhed) [inline spec/lib/rev/post_inputs_spec.rb]: Shouldn't both assertions use must_equal?

**Comment by tparnell** [comment]
This fix is only partially correct.  Check out: http://stackoverflow.com/questions/93551/how-to-encode-the-filename-parameter-of-content-disposition-header-in-http/6745788#6745788 and our implementation in HttpExtensions.EncodedFilenameDisposition

**Comment by denmarkin** [comment]
@tparnell there are no browsers to take care of. If MVC parses the string, that's all we need. We control the parsing.

See the test below 
image

**Comment by tparnell** [comment]
:shipit:

**Pull request closed without merging**